src/controllers/user/menu.js

```diff
@@ -0,0 +1,58 @@
+define(['apphost', 'connectionManager', 'layoutManager', 'listViewStyle', 'emby-button'], function(appHost, connectionManager, layoutManager) {
```

Comment:
Where did this file come from?

Reply:
https://github.com/jellyfin/jellyfin-web/blob/master/src/controllers/user/menu/index.js

Reply:
:+1: I didn't see this branch was outdated until I finished reviewing.